dlt/destinations/insert_job_client.py

```diff
@@ -72,6 +72,33 @@ def _insert(self, qualified_table_name: str, file_path: str) -> Iterator[List[st
                         else:
                             # Replace the , with ;
                             insert_sql.append("".join(chunk).strip()[:-1] + ";\n")
+                elif max_rows >= 1000:
+                    # This part breaks the multiple values in an insert statement into individual insert statements 
```

Comment:
I'm assuming you got this from here: https://stackoverflow.com/questions/36141006/how-to-insert-multiple-rows-into-sql-server-parallel-data-warehouse-table. If that is the case, it is always nice to leave a stackoverflow link in the comments to other devs can understand what is going on here.

Reply:
Also, why is this only needed for max_rows >= 1000? Shouldn't this always fail if we don't change to the union all approach?